### ISSUE TITLE
Add streaming profile to NS

### ISSUE THREAD

**Issue by rubensworks** [w3c/json-ld-wg#139]
The JSON-LD streaming note introduces a [streaming profile](https://w3c.github.io/json-ld-streaming/#streaming-profile): `http://www.w3.org/ns/json-ld#streaming`.

Can an entry for this be added to the JSON-LD namespace document https://www.w3.org/ns/json-ld ?

I'm asking first before opening a PR because this streaming profile comes from a note, and not an official recommendation.

**Comment by gkellogg** [comment]
Shouldn't be a problem adding to the namespace. I don't think it really takes WG approval, but @iherman would know the formalities. I think a PR is appropriate.

**Comment by iherman** [comment]
Indeed, adding to the namespace should be possible. I would think it is better, actually, to get an explicit WG approval (which should not be a problem) and a PR would be welcome indeed...
